/cam_port_manager:
  ros__parameters:
    # CatureNode Params
    # save_path: "/home/sonia/ssd/camera_data/"
    cam_ids:
      - 19208686
      - 19208682
    cam_aliases:
      - front
      - bottom
    default_cam_id: 19208686

    # Camera info message details
    image_height: 400
    image_width: 600
    distortion_model: plumb_bob
    binning: 1
    distortion_coeffs:
      front: [-0.021141875266089592, -0.3733872931278025, 2.385982550579459, 3.2824571732099725]
      bottom: [-0.021141875266089592, -0.3733872931278025, 2.385982550579459, 3.2824571732099725]

    #specified as [fx  0 cx 0 fy cy 0  0  1]
    intrinsic_coeffs:
      front: [1886.9232141485886, 0.0, 604.7214878709341, 0.0, 1886.6668765711668, 493.47726714719823, 0.0, 0.0, 1.0]
      bottom: [1886.9232141485886, 0.0, 604.7214878709341, 0.0, 1886.6668765711668, 493.47726714719823, 0.0, 0.0, 1.0]

    rectification_coeffs:
      front: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
      bottom: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]

    projection_coeffs:
      front: [913.700317, 0.000000, 953.448302, 0.000000, 0.000000, 1063.296631, 777.871993, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
      bottom: [913.700317, 0.000000, 953.448302, 0.000000, 0.000000, 1063.296631, 777.871993, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]